src/BuildIt.Forms/Samples/BuildIt.Forms.Sample.UWP/BuildIt.Forms.Sample.UWP.csproj

```diff
@@ -191,6 +191,9 @@
     <PackageReference Include="Plugin.Permissions">
       <Version>3.0.0.12</Version>
     </PackageReference>
+    <PackageReference Include="Xamarin.FFImageLoading.Forms">
+      <Version>2.4.3.840</Version>
+    </PackageReference>
     <PackageReference Include="Xamarin.Forms">
       <Version>3.1.0.583944</Version>
```

Comment:
Can we bump XF version to 3.1.0.697729 whilst we're making changes

Reply:
done